src/main/java/List.java

```diff
@@ -0,0 +1,221 @@
+public class List {
+    public static final String INVALID_EVENT_INPUT_MESSAGE = "event <event name> /from <start date/time> /end <end date/time>";
+    public static final String INVALID_DEADLINE_INPUT_MESSAGE = "deadline <deadline name> /by <deadline>";
+    public static final String INVALID_TODO_INPUT_MESSAGE = "todo <task name>";
+    public static final int DEADLINE_WORD_LEN = 8;
+    public static final int INPUT_SPACE_BUFFER = 2;
+    public static final int TODO_WORD_LEN = 4;
```

Comment:
maybe it would be better to use `input.contains("something")` rather than using the length of the comment. Anyway it still works